.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     docker:
       - image: cimg/node:16.14.0
       - image: rethinkdb:2.4.0
-      - image: circleci/postgres:12.3
+      - image: postgres:12.3
```

Comment:
See #6182. I also opened this up against the default branch to clear the CircleCI deprecation warning.